test/new-e2e/tests/agent-platform/install/installparams/install_params.go

```diff
@@ -51,6 +52,13 @@ func WithMajorVersion(majorVersion string) Option {
 	}
 }
 
+// WithInstallPythonThirdPartyDeps set the INSTALL_PYTHON_THIRD_PARTY_DEPS environment variable
```

Comment:
Example of an outdated comment. We are doing it differently now.

Reply:
nice catch, yes I'll remove the method and the param.